src/AppMap.d.ts

```diff
@@ -0,0 +1,198 @@
+namespace AppMap {
```

Comment:
Do you think these types could or should replace the types defined in the models package? How do we reconcile the two? Do we need to? It would seem nice.

Reply:
I believe so! I wanted to start with a clean slate to more easily uncover any problems and inconsistencies (https://github.com/getappmap/appmap/issues/43 suggests it was a good call...), but it is definitely my hope and intention to reconcile these. Perhaps create a separate package for types (or add them to https://github.com/getappmap/appmap ?) , since appmap-node doesn't really need to pull in *models*.